### ISSUE TITLE
Installing addon does not work when using yarn

### ISSUE THREAD

**Issue by rbarbey** [crodriguez1a/ember-sort-filter-table#35]
I installed this addon using the typical `ember install ember-sort-filter-table` command which in turn caused ember-cli to install the addon using yarn v0.27.5. Next, I tried to build and serve my app using `ember serve` but I got a Broccoli error:

```
The Broccoli Plugin: [BroccoliMergeTrees: Addon#compileAddon(ember-sort-filter-table) ] failed with:
ReferenceError: Unknown plugin "transform-decorators-legacy" specified in "base" at 0, attempted to resolve relative to "ember-sort-filter-table/components"
    at /Users/robert/Development/zoo-monitor-app/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (<anonymous>)
    [snip]
```

I checked the `package.json` of the addon and found a dev dependency called `babel-plugin-transform-decorators-legacy`. So installed that manually using yarn. Next, I got a very similar error complaining about `transform-class-properties` not being present. After installing that as well, I got the app to build.

I'm not familiar with publishing Ember addon, but I'm wondering whether the two dependencies (`babel-plugin-transform-class-properties` and `babel-plugin-transform-decorators-legacy`) need to be moved to the `dependencies` section of your package.json.

Let me know if I can help in any way. Thanks!

**Comment by crodriguez1a** [comment]
@rbarbey Good catch! You're right, [these](https://github.com/crodriguez1a/ember-sort-filter-table/blob/master/package.json#L28) dependencies are going to be required for decorators. I'll release a patch. Btw, if you get to this before I do please feel free to submit a PR.

**Comment by rbarbey** [comment]
@crodriguez1a On it 😄

**Comment by crodriguez1a** [comment]
Hey, thanks @rbarbey for turning that around so quickly. I shouldn't have missed that. Just published tag [0.3.1](https://github.com/crodriguez1a/ember-sort-filter-table/releases/tag/0.3.1) with the fix.
